source/reporting/index.html.md.erb

```diff
@@ -0,0 +1,113 @@
+---
+title: Reporting 
+weight: 70
+---
+
+# Reporting 
+
+You can extract reporting information from GOV.UK Pay manually, or via the
+API.
+
+## Manual reporting
+
+You can sign in to your [GOV.UK Pay admin
+account](https://selfservice.payments.service.gov.uk/login) and export
+transaction information in a CSV file. 
+
+For a given account, select __Transactions__ in the __Dashboard__ page.
+
+You can filter transactions using the available search criteria, and then
+select __Download all transaction details as a CSV file__. 
+
+You can process CSV files using spreadsheet software.
+
+## Reporting via the GOV.UK Pay API 
+
+Before reading this section, you should read the section describing [Making
+payments](/making_payments). 
+
+You can use the GOV.UK Pay API to export data automatically into your finance
```

Comment:
Pay isn’t really exporting the data into the finance systems (not directly, anyway). It’s more that one can develop software to get the data from the API and import it into a finance system.